package.json

```diff
@@ -123,6 +123,7 @@
     "@graphql-codegen/typescript-apollo-client-helpers": "^2.1.10",
     "@graphql-codegen/typescript-operations": "^2.2.4",
     "@graphql-codegen/typescript-react-apollo": "^3.2.5",
+    "@playwright/test": "^1.37.1",
```

Comment:
I suggest putting playwright into a QA or CI dependency group so they don't get installed locally as it will cause to have unneeded dev dependencies (more storage space used, slower installs, more dependencies to conflict, etc).